apps/web/src/components/DepositAndBorrow/Borrow/CurrentBorrowTable.tsx

```diff
@@ -0,0 +1,48 @@
+import { BoxLayout, Button, CellProps, Table } from '@bloxifi/ui'
```

Comment:
@aleksaToljic This is not completed. I just created a component for now because I haven't fetched the data yet.

Reply:
CurrentBorrowTable will be completed after Borrow modal